ports/butterflysoup/port.json

```
@@ -0,0 +1,36 @@
+{
+  "version": 3,
+  "name": "butterflysoup.zip",
+  "items": [
+    "Butterfly Soup.sh",
+    "butterflysoup"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Butterfly Soup",
+    "porter": [
+      "all_yall"
+    ],
+    "desc": "A visual novel about gay asian girls playing baseball and falling in love.",
+    "desc_md": "A visual novel about gay asian girls playing baseball and falling in love.",
+    "inst": "Get the game from itch.io here; https://brianna-lei.itch.io/butterfly-soup (Please consider buying!). Put the contents of the `game` directory into the gamefiles directory in the port.",
+    "inst_md": null,
```

You can add a proper url here.
```suggestion
    "inst_md": "Get the game from [Brianna Lei's itch.io](https://brianna-lei.itch.io/butterfly-soup). Place the contents of the `game/` directory into the `port/butterflysoup/gamefiles/` directory.",
```

---

Does this replace the `inst` field or is it along with?

---

`"inst"` gets shown in the PortMaster app on device, `"inst_md"` supports markdown and is used on the PortMaster website so you can add things like embedded urls.